app/managers.py

```diff
@@ -0,0 +1,38 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.sqlite3")
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} "
+            "(first_name, last_name) VALUES (?, ?) ",
+            (first_name, last_name),
+        )
+        self._connection.commit()
+
+    def all(self) -> Actor:

```

Comment:
The return type of the `all` method is incorrect. It should return a list of `Actor` objects, not a single `Actor`. Consider changing the return type annotation to `List[Actor]`.